test/e2e/cloud/gcp/ha_master.go

```diff
@@ -171,7 +171,7 @@ var _ = SIGDescribe("HA-master [Feature:HAMaster]", func() {
 		e2eskipper.SkipUnlessProviderIs("gce")
 		c = f.ClientSet
 		ns = f.Namespace.Name
-		framework.ExpectNoError(waitForMasters(framework.TestContext.CloudConfig.MasterName, c, 1, 10*time.Minute))
+		framework.ExpectNoError(waitForControlPlanes(framework.TestContext.CloudConfig.MasterName, c, 1, 10*time.Minute))
```

Comment:
[1]
`TestContext.CloudConfig.MasterName` should be safe to change, given the TestContext struct gives no deprecation guarantees to users.